internal/hardware/validator.go

```diff
@@ -46,6 +46,10 @@ type ValidatorCfg struct {
 	MaximumAllowedTimeDiffMinutes    int64                        `envconfig:"HW_VALIDATOR_MAX_TIME_DIFF_MINUTES" default:"4"`
 	InstallationDiskSpeedThresholdMs int64                        `envconfig:"HW_INSTALLATION_DISK_SPEED_THRESHOLD_MS" default:"10"`
 	VersionedRequirements            VersionedRequirementsDecoder `envconfig:"HW_VALIDATOR_REQUIREMENTS" default:"[]"`
+	MaxNetworkLatencyMsMaster        float64                      `envconfig:"HW_VALIDATOR_MAX_NETWORK_LATENCY_MS_MASTER" default:"100"`
+	MaxPacketLossPercentageMaster    float64                      `envconfig:"HW_VALIDATOR_MAX_PACKET_LOSS_PERCENTAGE_MASTER" default:"0"`
+	MaxNetworkLatencyMsWorker        float64                      `envconfig:"HW_VALIDATOR_MAX_NETWORK_LATENCY_MS_MASTER" default:"1000"`
+	MaxPacketLossPercentageWorker    float64                      `envconfig:"HW_VALIDATOR_MAX_PACKET_LOSS_PERCENTAGE_MASTER" default:"0"`
```

Comment:
Before setting these values, we need to look at customer data and understand what percentage of clusters would have been declared uninstallable.  We do not want to be overly strict at this point until we support profiles.

Reply:
sounds good. I'm thinking of leaving it as 0 by default to avoid causing issues in existing clusters. These values can be changed per deployment later on using @jakub-dzon new code. WDYT?

Reply:
on a second thought. I will disable these validations for now, pending on evaluating the results of the latency and packet loss in current production environments.

Reply:
Done. @avishayt these 2 validations are now disabled by default. I updated the subsystems tests and removed the extra code that was added to allow the tests to pass when these validations were enabled.